README.md

```diff
@@ -44,14 +44,21 @@ The priorities are all about the stores' user experience and retailer satisfacti
 
 1. Performance - Great score on Lighthouse and Core Web Vitals
 2. Stability - E2E tests are complete and make sure the critical paths are covered
-3. No fork - Features should be delivered by dependencies and not by a store fork
+3. SEO - Stores should pass on all aspects of Google search console, like complete structured data and web vitals
 4. Feature completion - Enable you to craft your custom ecommerce experience
 
-## Why Jamstack and Gatsby?
+## Basic Architecture
 
-Jamstack focuses on performance and stability, which is part of our core philosophy.
+FastStore core is composed of two main packages:
+1. [`store-ui`](https://storeui.netlify.app/): A performant ecommerce ready component library
+2. `store-sdk`: An sdk for ecommerce to solve your basic ecommerce needs, like cart, pixel managment, session etc.
+
+These packages have bidings to popular frameworks, like Gatsby. These bindings are listed in this monorepo and help you create stores with Multiregion, Internationalization etc.
+We want to support other frameworks, like Next.JS and Create React App in the future
```

Comment:
I'm not sure about listing Create React App as a framework. I would separate Next.JS from it. Something along the lines of: "We also want to support other React frameworks in the future, such as Next.JS, or even React applications bootstrapped using Create React App". 
Or not even mentioning Create React App at all 🤔  What do you think?